mappings/net/minecraft/entity/AreaEffectCloudEntity.mapping

```diff
@@ -33,6 +33,7 @@ CLASS ahx net/minecraft/entity/AreaEffectCloudEntity
 	METHOD f getRadius ()F
 	METHOD i getColor ()I
 	METHOD k getParticleType ()Lgd;
+	METHOD l isWaiting ()Z
 	METHOD m getDuration ()I
 	METHOD u getOwner ()Laih;
 	METHOD z updateColor ()V
```

Comment:
While you are at it, `setPotionEffect` should be renamed to `addPotionEffect`

Reply:
We don't have `PotionEffect`. We only have `StatusEffect`

Reply:
Oh. Indeed. Sounds like there are a few names that need to be changed then.

Reply:
Done